content/reference/apis/standard.md

```diff
@@ -50,3 +50,4 @@ Both TextEncoder and TextDecoder support UTF-8 encoding/decoding.
 The URL API supports urls conforming to http and https schemes.
 
 [Go to the docs](https://developer.mozilla.org/en-US/docs/Web/API/URL)
+[Shimming the URL class](/tooling/wrangler/webpack/#shimming-globals)
```

Comment:
```suggestion

_Note: For security reasons, our URL polyfill implementation differs slightly from the standard. If you're having encoding issues, you can [shimmy the URL class using webpack](/tooling/wrangler/webpack/#shimming-globals)_
```

Reply:
i'm +/- 0 on this phrasing - what if someone doesn't know who/what whatwg is? this is kind of an edge-case thing so it may just have to end up being phrased this way, i'm unsure 🤷‍♂ 

btw just a heads-up, the syntax as it is will render both these links on the same line. @victoriabernard92's suggestion will fix that 👍

Reply:
ah awesome. thank you. yah i'm not stoked about the phrasing (i edited it), i'd like as many folks to weigh in as possible before i commit.

Reply:
Also it's not guarantee that we follow the whatwg standard

Reply:
I think less is more: 

```suggestion
_Note: Our URL implementation differs slightly from standards. To use an external implementation, you can [shim the URL class using webpack](/tooling/wrangler/webpack/#shimming-globals)._
```